src/StripeTests/Services/OAuth/OAuthTokenServiceTest.cs

```diff
@@ -0,0 +1,89 @@
+namespace StripeTests
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net;
+    using System.Net.Http;
+    using System.Threading.Tasks;
+
+    using Stripe;
+    using Xunit;
+
+    public class OAuthTokenServiceTest : BaseStripeTest
+    {
+        private const string OrderId = "or_123";
+
+        private readonly OAuthTokenService service;
+        private readonly OAuthTokenCreateOptions createOptions;
+        private readonly OAuthTokenDeauthorizeOptions deauthorizeOptions;
+
+        public OAuthTokenServiceTest(MockHttpClientFixture mockHttpClientFixture)
+            : base(mockHttpClientFixture)
+        {
+            this.service = new OAuthTokenService();
+
+            this.createOptions = new OAuthTokenCreateOptions
+            {
+                Code = "this_is_an_authorization_code",
+                GrantType = "authorization_code",
+            };
+
+            this.deauthorizeOptions = new OAuthTokenDeauthorizeOptions
+            {
+                StripeUserId = "acct_test_deauth",
```

Comment:
This should have the `client_id=ca_123` per https://stripe.com/docs/connect/standard-accounts#revoked-access